pages/common/sdiff.md

```diff
@@ -0,0 +1,23 @@
+# sdiff
+
+> Compare the differences and/or merge between two files.
+
+- Compare 2 files:
+
+`sdiff {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring all tabs and whitespace:
+
+`sdiff -W {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring whitespace at the end of lines:
+
+`sdiff -Z {{path/to/first_file}} {{path/to/second_file}}`
+
+- Case insensitively compare 2 files:
```

Comment:
Compare in a case-insensitive manner:

Reply:
I just change this as @sbrl suggest. Personally I like 'Compare 2 files in a case-insensitive manner'.

What do you think ?

Reply:
Oh I see. "Case insensitively compare" sounded a bit odd to me.

Actually "case insensitively" is just used twice in the repo. Rest are all "case insensitive <verb>". Not a big deal either way.

Reply:
I agree. I'll use 'Compare 2 files in a case-insensitive manner' then :D